docs/modules/relabel.md

```diff
@@ -10,7 +10,7 @@
 This module allows you to modify the event metadata or drop the event at all.
 It uses native Prometheus `relabel_config` syntax. In this case metadata is referred as labels.
 See [the upstream documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config)
-for more info.
+for more info. Referenced metadata keys needs to be a valid Prometheus' label name. Use [eventMetadataRename](event_metadata_rename.md) if you need to sanitize them.
```

Comment:
```suggestion
for more info. Referenced metadata keys needs to be a valid Prometheus' label name.
```

Prefer to skip reference to experimental modules.

Reply:
Fixed within https://github.com/seznam/slo-exporter/pull/60/commits/5028e58871b4b43d5bff5cb4ec1f0ed009fac9e0